site/en/docs/webstore/best_practices/index.md

```diff
@@ -71,35 +71,82 @@ The developer console lets you specify a category for each extension. Choose the
 
-Fun
+Education
+: Extensions that teach or aid in teaching, including language learning, note taking, teaching aids, sign-language instruction among others.
+
+Entertainment
+: These extensions are designed for fans of sports, music television, and cinema.
+
+Functionality & UI
+: Extensions that enhance the Chrome user interface, such as tab managers, shortcut managers, and app launchers.
+
+Games
+: Extensions providing a wide array of desktop and archade style games.
+
+Household
+: Extenions for helping you around the house. This category includes, recipe savers and managers, budgeting, product research, and more.
+
+Just for Fun
 : These extensions are designed for entertainment. They can include games, interesting new tab backgrounds, quirky widgets, jokes, trivia, and more.
 
 News & Weather
 : These extensions keep users informed about current events and weather conditions. They can collect news from multiple sources, present real-time weather updates, notify breaking news, and more.
 
-Photos
-: These extensions provide tools for viewing, editing, organizing, and sharing photos. They may also offer features for capturing screenshots, image searching, and integrating with popular image hosting or editing services.
-
-Productivity
-: This is the most popular category. It provides tools to help users perform their tasks more efficiently. They could range from time trackers, tools to stay focused, to-do list managers, email organizers, document editors, and calendar utilities, among others.
-
-Search Tools
-: These extensions enhance the browser's search capabilities. For example, custom search engines, quick search shortcuts, automatic highlighters, etc.
+Privacy & Security
+: Extensions such as VPNs, passwords safes, and phishing deterance.
 
 Shopping
 : These extensions aim to enhance the online shopping experience. They might offer features like price comparison, coupon finders, reviews and ratings, wish list management, and more.
 
-Social & Communication
+Social Media & Networking
 : These extensions are designed to enhance social media platforms. They can integrate with services and offer features like easy sharing, notifications, status updates, and more.
 
-Sports
-: These extensions are designed for sports fans. They can provide live score updates, sports news, game reminders, manage sports fantasy leagues, and integrate with various sports platforms.
+Tools
+: Tools that don't fit into other categories
+
+Travel
+: Extensions for planning trips.
+
+Well-being
+: Extensions for self-help and personal improvement.
+
+Workflow & Planning
+: Extensions to help users perform their tasks more efficiently. They could range from time trackers, tools to stay focused, to-do list managers, email organizers, document editors, and calendar utilities, among others.
+
+In mid 2023 the categories changed. Most of the new categories match previous ones. Several were
```

Comment:
This intro should probably go to the top of the categories section

Reply:
I added a heading instead. This content needs to move out as soon as I find a better place for it.